### PR TITLE
Turned off Ctrl+C processing by Windows shell

### PR DESCRIPTION
**Pull request by brendandixon**
Signed-off-by: Brendan Dixon brendand@microsoft.com

### CODE REVIEW

**Comment by brendandixon** [comment]
This small change removes the bit causing the Windows shell to consume Ctrl+C. It appears to improve some other behavior as a by-product. No other guarantees. ;)

**Comment by jfrazelle** [comment]
Hey @brendandixon,

We were thinking of making RC6 before EOD, do you have any more changes other than this or should we wait til monday.

File: pkg/term/winconsole/console_windows.go [hunk 1]
```diff
@@ -1038,8 +1042,7 @@ func (term *WindowsTerminal) HandleInputSequence(fd uintptr, command []byte) (n
 }
 
 func marshal(c COORD) uintptr {
-	// works only on intel-endian machines
-	return uintptr(uint32(uint32(uint16(c.Y))<<16 | uint32(uint16(c.X))))
+	return uintptr(*((*DWORD)(unsafe.Pointer(&c))))
```
Comment (jfrazelle) [inline pkg/term/winconsole/console_windows.go]: does this work on more than intel-endian machines

**Comment by thaJeztah** [comment]
Does this one address https://github.com/docker/docker/issues/12158?

**Comment by jfrazelle** [comment]
can confirm it fixes #12158

**Comment by icecrime** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM

**Comment by jfrazelle** [comment]
thanks for fixing things so fast @brendandixon you rock!

**Comment by brendandixon** [comment]
Sorry. Went dark for a bit. Thank you for merging. You should proceed with your RC. Any other fixes will likely be too invasive for you at this time.

File: pkg/term/winconsole/console_windows.go (hunk 1 above)
Comment (brendandixon) [inline pkg/term/winconsole/console_windows.go]: Yes. In fact, the prior code was limited to Intel-style hardware. This approach is generic.

**Pull request merged**